README.md

```diff
@@ -2,8 +2,9 @@
 
 ## Requirements
 
-- Node 16+
-- Python 3.7+
+-   Node 18+
+-   pnpm 9+
+-   Python 3.7+
```

Comment:
Since we drop support for python 3.7, we shall update it to Python 3.8+

Reply:
also node needs to upgrade to 20?